internal/commands/result.go

```diff
@@ -343,7 +343,7 @@ func createRequestedScannersResponse(scanTypes map[string]string, scanResponseMo
 func createAllFailedScannersResponse(scanResponseModel *wrappers.ScanResponseModel) []ScannerResponse {
 	var results []ScannerResponse
 	for i := range scanResponseModel.StatusDetails {
-		if scanResponseModel.StatusDetails[i].Status == wrappers.ScanFailed && scanResponseModel.StatusDetails[i].Name != General {
+		if scanResponseModel.StatusDetails[i].Status == wrappers.ScanFailed {
```

Comment:
so we will always add the general and it's approved?

Reply:
can you have a test for this too?

Reply:
Yes, because we need to display issues that are not engine-specific.
tested were added in unit tests.